kubernetes/adapter.go

```diff
@@ -183,9 +183,9 @@ func (a *Adapter) newIngress(typ IngressType, metadata kubeItemMetadata, host st
 		shared = false
 	}
 
-	ipAddressType := aws.IPAddressTypeIPV4
-	if getAnnotationsString(annotations, ingressALBIPAddressType, "") == aws.IPAddressTypeDualstack {
-		ipAddressType = aws.IPAddressTypeDualstack
+	ipAddressType := aws.IPAddressTypeDualstack
```

Comment:
If you plan to promote this, lets make it backwards compatible by having a flag for the default address type.